docs/source/eFeatures.rst

```diff
@@ -573,6 +603,296 @@ The burst detection can be fine-tuned by changing the setting strict_burst_facto
+
+This implementation does not assume that every spike belongs to a burst.
+
+The first spike is ignored by default. This can be changed by setting ignore_first_ISI to 0.
+
+The burst detection can be fine-tuned by changing the setting strict_burst_factor. Defalt value is 2.0.
+
+- **Required features**: postburst_fast_ahp_indices, burst_end_indices, peak_indices
+- **Units**: mV
+- **Pseudocode**: ::
+
+    interburst_30percent_values = []
+    for i, postburst_fahp_i in enumerate(postburst_fahpi):
+        if i < len(burst_endi) and burst_endi[i] + 1 < len(peaki):
+            time_interval = t[peaki[burst_endi[i] + 1]] - t[postburst_fahp_i]
+            time_at_30percent = t[postburst_fahp_i] + time_interval * 0.30
+            index_at_30percent = numpy.argwhere(t >= time_at_30percent)[0][0]
+            interburst_30percent_values.append(v[index_at_30percent])
+
+`LibV5`_ : interburst_40percent_values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Voltage value after 40% of the interburst duration after the fast AHP.
+
+This implementation does not assume that every spike belongs to a burst.
+
+The first spike is ignored by default. This can be changed by setting ignore_first_ISI to 0.
+
+The burst detection can be fine-tuned by changing the setting strict_burst_factor. Defalt value is 2.0.
+
+- **Required features**: postburst_fast_ahp_indices, burst_end_indices, peak_indices
+- **Units**: mV
+- **Pseudocode**: ::
+
+    interburst_40percent_values = []
+    for i, postburst_fahp_i in enumerate(postburst_fahpi):
+        if i < len(burst_endi) and burst_endi[i] + 1 < len(peaki):
+            time_interval = t[peaki[burst_endi[i] + 1]] - t[postburst_fahp_i]
+            time_at_40percent = t[postburst_fahp_i] + time_interval * 0.4
+            index_at_40percent = numpy.argwhere(t >= time_at_40percent)[0][0]
+            interburst_40percent_values.append(v[index_at_40percent])
+
+`LibV5`_ : interburst_60percent_values
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Voltage value after 60% of the interburst duration after the fast AHP.
+
+This implementation does not assume that every spike belongs to a burst.
+
+The first spike is ignored by default. This can be changed by setting ignore_first_ISI to 0.
+
+The burst detection can be fine-tuned by changing the setting strict_burst_factor. Defalt value is 2.0.
+
+- **Required features**: postburst_fast_ahp_indices, burst_end_indices, peak_indices
+- **Units**: mV
+- **Pseudocode**: ::
+
+    interburst_60percent_values = []
+    for i, postburst_fahp_i in enumerate(postburst_fahpi):
+        if i < len(burst_endi) and burst_endi[i] + 1 < len(peaki):
+            time_interval = t[peaki[burst_endi[i] + 1]] - t[postburst_fahp_i]
+            time_at_60percent = t[postburst_fahp_i] + time_interval * 0.6
+            index_at_60percent = numpy.argwhere(t >= time_at_60percent)[0][0]
+            interburst_60percent_values.append(v[index_at_60percent])
```

Comment:
Can these documentations be grouped similar to  ` Python efeature_ : inv_first_ISI, inv_second_ISI, inv_third_ISI, inv_fourth_ISI, inv_fifth_ISI, inv_last_ISI` to avoid copy pastes in the docs and the pseudocode? The pseudo code can have a variable for the percentage constant  perhaps?

Reply:
Good catch. Done in latest commit

Reply:
done in latest commit